packages/components/src/internal/components/ontology/OntologyLookupOptions.tsx

```diff
@@ -69,18 +74,42 @@ export class OntologyLookupOptions extends PureComponent<Props, State> {
     };
 
     onOntologyFieldChange = (evt: any): void => {
-        this.onChange(evt.target.id, evt.target.value);
+        const { domainIndex, onMultiChange } = this.props;
+        const { id, value } = evt.target;
+        let changes = List<IFieldChange>([{ id, value }]);
+        changes = changes.push({
+            id: createFormInputId(DOMAIN_FIELD_ONTOLOGY_SUBTREE_COL, domainIndex, getIndexFromId(id)),
+            value: undefined,
+        });
+        onMultiChange(changes);
+    };
+
+    onOntologySubtreeChange = async (id: string, path: PathModel): Promise<void> => {
+        let subtreePath;
+        if (path?.path) {
+            const parents = await fetchParentPaths(path?.path);
```

Comment:
The null check is done above
```suggestion
            const parents = await fetchParentPaths(path.path);
```

Reply:
Done